org.jacoco.core.test.validation.groovy/pom.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (c) 2009, 2018 Mountainminds GmbH & Co. KG and Contributors
+   All rights reserved. This program and the accompanying materials
+   are made available under the terms of the Eclipse Public License v1.0
+   which accompanies this distribution, and is available at
+   http://www.eclipse.org/legal/epl-v10.html
+
+   Contributors:
+      Andres Almiray - initial API and implementation
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.jacoco</groupId>
+        <artifactId>org.jacoco.core.test.validation</artifactId>
+        <version>0.8.3-SNAPSHOT</version>
+        <relativePath>../org.jacoco.core.test.validation</relativePath>
+    </parent>
+
+    <artifactId>org.jacoco.core.test.validation.groovy</artifactId>
+
+    <name>JaCoCo :: Test :: Core :: Validation Groovy 2</name>
+
+    <properties>
+        <gmaven.version>1.6</gmaven.version>
+        <groovy.version>2.5.3</groovy.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>org.jacoco.core.test</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.codehaus.groovy</groupId>
+            <artifactId>groovy-all</artifactId>
+            <version>${groovy.version}</version>
+            <type>pom</type>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.22.0</version>
+                <configuration>
+                    <testSourceDirectory>${project.build.sourceDirectory}</testSourceDirectory>
+                    <testClassesDirectory>${project.build.outputDirectory}</testClassesDirectory>
```

Comment:
`testSourceDirectory` and `testClassesDirectory` already specified in parent pom, so should be removed.

Reply:
Reason to having this block is to apply version `2.22.0` of the surefire plugin, however I've removed this block as the `groovy-all` dependency was changed to just `groovy`.

Reply:
I believe that Maven takes configuration from parent even in case of version change.

Reply:
Does it? I think that the parent can define `<inherited>true</inherited>` in the plugin block to allow children to override values, but I may be wrong. Anyway, old version and smaller groovy dep are now in place.

Reply:
can define `<inherited>false</inherited>` which is not the case for our parent that uses [default `<inherited>true</inherited>`](https://maven.apache.org/guides/mini/guide-configuring-plugins.html#Using_the_inherited_Tag_In_Build_Plugins) 😉 anyway, thanks!